redbot/message/status.py

```diff
@@ -340,6 +342,13 @@ class STATUS_UNSUPPORTED_MEDIA_TYPE(Note):
     text = """\
  """
 
+class STATUS_IM_A_TEAPOT(Note):
+    category = categories.GENERAL
+    level = levels.INFO
+    summary = "Teapot is requested to brew coffee."
```

Comment:
This needs to be more descriptive of the situation; this status code isn't a part of HTTP, so it should say that.

Also, there needs to be a text (as per other Notes) explaining in more detail. The abstract of draft-nottingham-thanks-larry is a good starting point.